### ISSUE TITLE
 TypeError: Cannot read property 'getRange' of undefined	at Cache.showBundlerErrors

### ISSUE THREAD

**Issue by wtjones** [42Crunch/vscode-openapi#123]
This started recently on 1.55.0 extension 4.3.1. 

I can preview as expected up until a few minutes until the hot reload of swagger UI stops working. If I run the open preview command I get "Failed to generate preview, check OpenAPI file for errors." There are no errors in the file. Closing out vscode allows the extension to work again for a time.

Things I tried:
- Close/reopen the yaml or switch to another
- Reinstalling the extension
- Disabling spectre lint in case of conflict

The developer console show this:

```
Stack trace: TypeError: Cannot read property 'getRange' of undefined	at Cache.showBundlerErrors (c:\Users\Anon\.vscode\extensions\42crunch.vscode-openapi-4.3.1\out\cache.js:350:43)	at Cache.<anonymous> (c:\Users\Anon\.vscode\extensions\42crunch.vscode-openapi-4.3.1\out\cache.js:166:30)	at Generator.next (<anonymous>)	at fulfilled (c:\Users\Anon\.vscode\extensions\42crunch.vscode-openapi-4.3.1\out\cache.js:28:58)
```

**Comment by ak1394** [comment]
Thanks for reporting! That's certainly looks like a regression introduced by a recent changes. 

When that happens to you, does it happen when you edit specific file(s)?
Do you use external references I.e. $ref: "foo.yaml#/components/schema/bar" ? 
Can you share the OpenAPI file?

**Comment by wtjones** [comment]
I'm not sure if it affects only certain files but I should be able to let you know soon.
I do not use external references.
Unfortunately I cannot share it.

**Comment by ak1394** [comment]
Thanks for the info! Knowing that you're not using external refs is already helpful. I would expect a new release with the fix some time at the start of the next week, in the meantime you can downgrade to an older version which probably should fix your issue.

**Comment by ak1394** [comment]
The issue should be fixed in the new v4.4.0 release. Please let me know how it works for you.
